GrandCentralBoard/Widgets/Bonus/BonusWidget.swift

```diff
@@ -11,15 +11,20 @@ final class BonusWidget: WidgetControlling {
     private let widgetView: BonusWidgetView
     let sources: [UpdatingSource]
     let bubbleResizeDuration: NSTimeInterval
+    let numberOfBubbles: Int
 
-    init(sources: [UpdatingSource], bubbleResizeDuration: NSTimeInterval) {
+    init(sources: [UpdatingSource], bubbleResizeDuration: NSTimeInterval, numberOfBubbles: Int) {
         self.widgetView = BonusWidgetView.fromNib()
         self.sources = sources
         self.bubbleResizeDuration = bubbleResizeDuration
+        self.numberOfBubbles = numberOfBubbles
     }
 
     var view: UIView {
-        return widgetView
+        let viewModel = WidgetTemplateViewModel(title: "BONUSLY", subtitle: "LAST \(numberOfBubbles) PEOPLE", contentView: widgetView)
+        let layoutSettings = WidgetTemplateLayoutSettings(contentMargin: UIEdgeInsetsZero)
+        let templateView = WidgetTemplateView.viewWithViewModel(viewModel, layoutSettings: layoutSettings)
```

Comment:
We pass this `view` to parent authority which holds `WidgetControlling`. We're not guaranteed that it calls this `view` getter only once, so it will be safer to construct `templateView` initially (f.e. in initializer) and only return the reference from here.